server/utils/nunjucksSetup.ts

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-param-reassign */
+import nunjucks from 'nunjucks'
+import express from 'express'
+import * as pathModule from 'path'
+
+const production = process.env.NODE_ENV === 'production'
+
+export default function nunjucksSetup(app: express.Express, path: pathModule.PlatformPath): void {
+  app.engine('njk', nunjucks.render)
+  app.set('view engine', 'njk')
+
+  app.locals.asset_path = '/assets/'
+  app.locals.applicationName = 'Check my diary'
+
+  // Cachebusting version string
+  if (production) {
+    // Version only changes on reboot
+    app.locals.version = Date.now().toString()
+  } else {
+    // Version changes every request
+    app.use((req, res, next) => {
+      res.locals.version = Date.now().toString()
+      return next()
+    })
+  }
+
+  const njkEnv = nunjucks.configure(
+    [
+      path.join(__dirname, '../../server/views'),
+      'node_modules/govuk-frontend/',
+      'node_modules/govuk-frontend/components/',
+      'node_modules/@ministryofjustice/frontend/',
+      'node_modules/@ministryofjustice/frontend/moj/components/',
+    ],
+    {
+      autoescape: true,
```

Comment:
don't pass express in here, instead manually configure nunjucks at the top of the file so that it can co-exist with ejs